### PR TITLE
LAM-974 feat: dockerize machine for integration tests

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+# Prevent cache invalidation when we only change version numbers
+COPY ["package.json", "package-lock.json", "./"]
+RUN npm version --allow-same-version --git-tag-version false --commit-hooks false 1.0.0
+RUN npm install
+COPY . ./
+RUN npm run build
+
+RUN cp ./mock_data/device_config.sample.json ./device_config.json
+RUN cp ./mock_data/licenses.sample.json ./licenses.json
```
Comment (siiky) [inline Dockerfile]: licenses are no longer used?

File: lib/update/updater.js [hunk 2]
```diff
@@ -11,7 +11,25 @@ var _ = require('lodash/fp')
...
+};
+
+console.error = function(){
+  var date = (new Date).toISOString() 
+  error(date, 'ERROR', ...arguments)
+};
+
+var consts =   {
+  port: 8000,
+  host: "updates.lamassu.is",
```
Comment (siiky) [inline lib/update/updater.js:29]: unused..?

File: build-scripts/build-docker.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+sed "s/WebSocket(.*)/WebSocket('ws:\/\/' + location.hostname + '\/ws')/g" ui/js/app.js > ui/js/docker-app.js
+sed "s/app.js/docker-app.js/g" ui/start.html > ui/start-docker.html
+sed -i "s/..\/node_modules\///g"  ui/start-docker.html
+sed "s/app.js/test-app.js/g" ui/start.html > ui/start-test.html
+sed -i "s/..\/node_modules\///g"  ui/start-test.html
```
Comment (siiky) [inline build-scripts/build-docker.sh]:
```
sed 's/app.js/docker-app.js/g; s/..\/node_modules\///g;' ui/start.html > ui/start-docker.html
sed 's/app.js/test-app.js/g; s/..\/node_modules\///g;' ui/start.html > ui/start-test.html
```

File: deploy/codebase/build.sh [hunk 4]
```diff
@@ -39,7 +39,6 @@ cp $UPDATESCRIPT $EXPORT_SCRIPT_DIR/updatescript.js
 
 # Codebase
 cp $MACHINE_DIR/*.js $TARGET_MACHINE_DIR
-cp $MACHINE_DIR/software_config.json $TARGET_MACHINE_DIR
 cp $MACHINE_DIR/licenses.json $TARGET_MACHINE_DIR
```
Comment (siiky) [inline deploy/codebase/build.sh]: licenses are no longer used?

**Review by siiky — CHANGES_REQUESTED** [review]

File: lib/update/updater.js (hunk 2 above)
Comment (RafaelTaranto) [reply lib/update/updater.js:29]: IDK why that was missing, added.

**Review by siiky — APPROVED** [review]

**Pull request merged**